domain/src/main/java/me/androidbox/domain/authentication/usecase/imp/LogoutUseCaseImp.kt

```diff
@@ -0,0 +1,14 @@
+package me.androidbox.domain.authentication.usecase.imp
+
+import me.androidbox.domain.authentication.ResponseState
+import me.androidbox.domain.authentication.usecase.AuthenticateUserUseCase
+import me.androidbox.domain.authentication.usecase.LogoutUseCase
+import javax.inject.Inject
+
+class LogoutUseCaseImp @Inject constructor(
+    private val authenticateUserUseCase: AuthenticateUserUseCase
+) : LogoutUseCase {
+    override suspend fun execute(): ResponseState<Unit> {
+        return authenticateUserUseCase.execute()
+    }
+}
```

Comment:
What is the point of this empty use case executing another use case?

Reply:
@philipplackner Will refactor this. I guess I am rushing to finish this. Will do this at a later time.